pkg/controller/pxc/backup.go

```diff
@@ -296,3 +297,105 @@ func (r *ReconcilePerconaXtraDBCluster) deletePITR(cr *api.PerconaXtraDBCluster)
 
 	return nil
 }
+
+var ErrNoBackups = errors.New("No backups found")
+
+func (r *ReconcilePerconaXtraDBCluster) getLatestSuccessfulBackup(ctx context.Context, cr *api.PerconaXtraDBCluster) (*api.PerconaXtraDBClusterBackup, error) {
+	bcpList := api.PerconaXtraDBClusterBackupList{}
+	if err := r.client.List(ctx, &bcpList, &client.ListOptions{Namespace: cr.Namespace}); err != nil {
+		return nil, errors.Wrap(err, "get backup objects")
+	}
+
+	if len(bcpList.Items) == 0 {
+		return nil, ErrNoBackups
+	}
+
+	latest := bcpList.Items[0]
+	for _, bcp := range bcpList.Items {
+		if bcp.Spec.PXCCluster != cr.Name || bcp.Status.State != api.BackupSucceeded {
+			continue
+		}
+
+		if latest.ObjectMeta.CreationTimestamp.Before(&bcp.ObjectMeta.CreationTimestamp) {
+			latest = bcp
+		}
+	}
+
+	// if there are no successful backups, don't blindly return the first item
+	if latest.Status.State != api.BackupSucceeded {
+		return nil, ErrNoBackups
+	}
```

Comment:
Could add in this case that we din't find successful backups (not backups at all)

Reply:
I don't get it, sorry. Do you mean returning a different error if no successful backups found?

Reply:
I mean as for me there is the difference: 
we don't have successful backups (not empty list)
we don't have backups (empty list)